lib/auth/webauthncli/fido2.go

```diff
@@ -77,7 +77,7 @@ var fidoNewDevice = func(path string) (FIDODevice, error) {
 
 // IsFIDO2Available returns true if libfido2 is available in the current build.
 func IsFIDO2Available() bool {
-	val, ok := os.LookupEnv("TSH_FIDO2")
```

Comment:
This is kind of a breaking change, isn't it? If so let's not forget to put this in the release notes for v10.

Reply:
This wasn't released yet, hence me changing it so unceremoniously.